src/Console/Database/FixHtmlEncodingCommand.php

```diff
@@ -298,4 +330,37 @@ private function doubleEncoding(string $input): string
         ];
         return preg_replace($pattern, $replace, $input);
     }
+
+    /**
+     * Fix double encoded HTML entities in old followups generated from Outlook
```

Comment:
Can probably also happen with other tools, as it was mainly due to a double encoding of email strings in our side.

```suggestion
     * Fix double encoded HTML entities in old followups generated
```